worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/CongestionController.java

```diff
@@ -332,4 +340,17 @@ public UserCongestionControlContext getUserCongestionContext(UserIdentifier user
   public BufferStatusHub getConsumedBufferStatusHub() {
     return consumedBufferStatusHub;
   }
+
+  private void updateQuota(ConfigService configService) {
```

Comment:
nit: configService is already a member variable

Reply:
Done.